Assets/LeapC/InvalidArm.cs

```diff
@@ -0,0 +1,93 @@
+﻿using UnityEngine;
+using System.Collections;
+
+namespace Leap
+{
+    public class InvalidArm : IArm
+    {
+        public Vector ElbowPosition
+        {
+            get { throw new System.NotImplementedException(); }
```

Comment:
Definitely a superior strategy to implement an invalid concept--but we chose to just deprecate the whole thing instead.